source/byte_buf.c

```diff
@@ -416,14 +416,20 @@ uint64_t aws_hash_byte_cursor_ptr_ignore_case(const void *item) {
     return aws_hash_array_ignore_case(cursor->ptr, cursor->len);
 }
 
-bool aws_byte_cursor_eq_byte_buf(const struct aws_byte_cursor *a, const struct aws_byte_buf *b) {
-    AWS_ASSERT(a && b);
-    return aws_array_eq(a->ptr, a->len, b->buffer, b->len);
+bool aws_byte_cursor_eq_byte_buf(const struct aws_byte_cursor *const a, const struct aws_byte_buf *const b) {
+    AWS_PRECONDITION(aws_byte_cursor_is_valid(a) && aws_byte_buf_is_valid(b));
+    bool rv = aws_array_eq(a->ptr, a->len, b->buffer, b->len);
+    AWS_POSTCONDITION(aws_byte_cursor_is_valid(a) && aws_byte_buf_is_valid(b));
+    return rv;
 }
 
-bool aws_byte_cursor_eq_byte_buf_ignore_case(const struct aws_byte_cursor *a, const struct aws_byte_buf *b) {
-    AWS_ASSERT(a && b);
-    return aws_array_eq_ignore_case(a->ptr, a->len, b->buffer, b->len);
+bool aws_byte_cursor_eq_byte_buf_ignore_case(
+    const struct aws_byte_cursor *const a,
+    const struct aws_byte_buf *const b) {
```

Comment:
Formatting change?

Reply:
The line size increased because of the new `*const` ones.